DevelopGitHub/GitHubExtension/GitHubExtension.Notes.Tests/TestsForValidator/NoteValidatorTests.cs

```diff
@@ -0,0 +1,37 @@
+﻿using FluentValidation.TestHelper;
+using GitHubExtension.Notes.WebApi.Validators;
+using Xunit;
+
+namespace GitHubExtension.Notes.Tests.TestsForValidator
+{
+    public class NoteValidatorTests
+    {
+        [Fact]
+        public void ShoulHaveErrorWhenBodyIsNull()
```

Comment:
There are positive tests below e.g. ShouldNotHaveErrorWhenBodyIsSpecified. I think it is better to rename it something like ValidModelWhenBodyIsSpecified. Made changes and commited.